5.5/contrib/etc/httpdconf.sed

```diff
@@ -1,7 +1,7 @@
 s/^Listen 80/Listen 0.0.0.0:8080/
 s/^User apache/User default/
 s/^Group apache/Group root/
-s%^DocumentRoot "/opt/rh/httpd24/root/var/www/html"%DocumentRoot "/opt/app-root/src"%
+s%^DocumentRoot "/opt/rh/httpd24/root/var/www/html"%#DocumentRoot "/opt/app-root/src"%
```

Comment:
sorry, it's too late here for me to be parsing regex with non-standard delimiters, i see what you're doing now. (commenting the line out).